drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -249,19 +249,9 @@
     font-weight: bold;
     text-align: center;
     text-transform: uppercase;
-    margin-top: auto;
-    margin-bottom: auto;
-}
-
-/*** IE9 alignment fix ***/
-.lt-ie10 .xblock--drag-and-drop .zone {
-    display: table;
-}
-
-.lt-ie10 .xblock--drag-and-drop .zone p {
-    display: table-cell;
-    vertical-align: middle;
-    text-align: center;
+    margin-top: auto;  /* Shield zone labels from LMS rules that change margin-bottom for <p> elements */
+                       /* to make sure they are centered vertically */
+    margin-bottom: auto !important;
```

Comment:
Note: Unless the rule you're trying to shield this block from is also using `!important`, it's better to override it by increasing the specificity of the selector for this one rule, i.e. using `.xblock--drag-and-drop .drag-container div.target div.zone p`, rather than using `!important`. Otherwise you can end up with a random mix of `!important` rules scattered throughout the CSS and theme CSS overrides, which is harder to maintain. And I think upstream frowns upon `!important` whenever not necessary.